### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dannylamb**
**JIRA Ticket**: (https://jira.duraspace.org/projects/ISLANDORA/issues/ISLANDORA-1817)
- Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)
# What does this Pull Request do?

Updates README documentation to explain that this module only works with JWPlayer 6.
# What's new?

README wordage.
# How should this be tested?

Read the README
# Additional Notes:
- Could this change impact execution of existing code? Nope :smiley: 
# Interested parties

Tagging @whikloj @manez @Islandora/7-x-1-x-committers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,19 +13,17 @@ This module requires the following modules/libraries:
...
 
-Install the JW Player available [here](http://www.longtailvideo.com/jw-player/download/) to your `sites/all/libraries` folder.
-
-Note: You must register for an account with JWPlayer to download the library.
+Install the JW Player available [here](https://github.com/jwplayer/jwplayer/releases/tag/v6.12.0) to your `sites/all/libraries` folder.  You will have to build it yourself.
 
 ### Notes
 
-This module has been updated to use jwplayer6. If you are updating this module for the first time since 22/01/2013 with an install predating that date your library will need to be updated.
+You must have prevoiusly registered for an account with JWPlayer when version 6 was being offered to legally use the library.  Registering now only licenses you to use version 7 as per JWPlayer's [terms of service](https://www.jwplayer.com/tos/).  If you are a new user looking to display video, please use the [Islandora Video.js](https://github.com/Islandora/islandora_videojs) module.
```
Comment (whikloj) [inline README.md]: "prevoiusly" typo

Comment (whikloj) [reply README.md]: "new user looking to display video"s
or
"new user looking to display video" content?

File: README.md [hunk 2]
```diff
@@ -13,19 +13,17 @@ This module requires the following modules/libraries:
...
+* [JW Player version 6](https://github.com/jwplayer/jwplayer/releases/tag/v6.12.0)
 
 ## Installation
 
 Install as usual, see [this](https://drupal.org/documentation/install/modules-themes/modules-7) for further information.
 
-Install the JW Player available [here](http://www.longtailvideo.com/jw-player/download/) to your `sites/all/libraries` folder.
-
-Note: You must register for an account with JWPlayer to download the library.
+Install the JW Player available [here](https://github.com/jwplayer/jwplayer/releases/tag/v6.12.0) to your `sites/all/libraries` folder.  You will have to build it yourself.
```
Comment (whikloj) [inline README.md]: Do we have any idea how to do this? Even a starting place might reduce the amount of support requests.

**Review by whikloj — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dannylamb) [reply README.md]: No idea. Never tried.  Le sigh... _grumbles_ I'll give it a whirl.

Comment (whikloj) [reply README.md]: Maybe just link to this
https://github.com/jwplayer/jwplayer

**Pull request merged**